sdk/typescript/introspector/scanner/abtractions/argument.ts

```diff
@@ -162,6 +162,14 @@ export class Argument {
     return this.formatDefaultValue(this.param.initializer.getText())
   }
 
+  /**
+   * Return true if the parameter is optional.
+   *
+   * A parameter is considered optional if he fits one of the following:
```

Comment:
Was just correcting the "he", but the rest can be removed:
```suggestion
   * A parameter is considered optional if:
```